### PR TITLE
fix: fix no pod create/execute rules

### PR DESCRIPTION
**Pull request by WitoDelnat**
This PR fixes bugs in KBP107 and KBP108

### CODE REVIEW

File: packages/validation/src/validators/practices/rules/KBP107-noPodCreate.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import {isClusterRole} from '../../custom/schemas/clusterrole.rbac.authorization
 import {isRole} from '../../custom/schemas/role.rbac.authorization.k8s.io.v1.js';
 import {NSA_RELATIONS} from '../../../taxonomies/nsa.js';
 
+const KIND_CREATES_PODS = ['pods', 'deployments', 'statefulsets', 'jobs', 'cronjobs'];
```
Comment (f1ames) [inline packages/validation/src/validators/practices/rules/KBP107-noPodCreate.ts:7]: Looking at the [specs here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#podtemplatespec-v1-core), it seems there are few more kinds which can create Pods:

image

AFAIU, all "Workloads APIs" can result in Pods creation:

image

Not 100% though, does it relates directly to this rule?

**Review by f1ames — CHANGES_REQUESTED** [review]
Looks good, just one clarification needed 👍

File: packages/validation/src/validators/practices/rules/KBP107-noPodCreate.ts (hunk 1 above)
Comment (WitoDelnat) [reply packages/validation/src/validators/practices/rules/KBP107-noPodCreate.ts]: Thanks I overlooked DaemonSet and ReplicaSet, ReplicationController is not really used afaik

**Review by f1ames — APPROVED** [review]
👍

**Pull request merged**